plugins/janus_streaming.c

```diff
@@ -575,6 +584,32 @@ void *janus_streaming_watchdog(void *data) {
 	return NULL;
 }
 
+static int janus_streaming_lookup_multicast_interface(const struct ifaddrs * ifas, const char * iface, janus_network_address * result) {
+	if(result) {
+		if(iface) {
+			janus_network_query_config q;
+			if(janus_prepare_network_device_query(iface, IPv4OnlyMatch | NameOnlyMatch, &q)) {
+				JANUS_LOG(LOG_ERR, "Unable to prepare query of network devices by name and/or IPv4 address for values matching: %s\n", iface);
+				return -1;
+			}
```

Comment:
Please put the else right after the closing bracket (same in other parts of the code where it happens too).